app/javascript/react/containers/RestaurantFormContainer.js

```diff
@@ -0,0 +1,296 @@
+      this.setState({ file: file })
+    } else {
+      this.setState({ message: 'You can only upload one photo per restaurant.'})
+    }
+  }
+
+
+  render() {
+
+    let errorDiv;
+    let errorItems;
+    if (Object.keys(this.state.errors).length > 0) {
+      errorItems = Object.values(this.state.errors).map(error => {
+        return(<li key={error}>{error}</li>)
+      })
+      errorDiv = <div className="alert-box alert round">{errorItems}</div>
+
+    }
+
+    let noticesDiv;
+    let noticesItems;
+    if (Object.keys(this.state.notices).length > 0) {
+      noticesItems = Object.values(this.state.notices).map(error => {
+        return(<li key={error}>{error}</li>)
+      })
+      noticesDiv = <div className="alert-box notice round">{noticesItems}</div>
+
+    }
+
+    return (
+      <div>
+      <h1>Restaurant Form</h1>
+      {noticesDiv}
+      {errorDiv}
+
+      <form className="callout" id="newRestaurantForm" onSubmit={this.handleFormSubmit}>
+      <TextField
+        label='Name'
+        name='name'
+        handlerFunction={this.handleNameUpdate}
+        value={this.state.name}
+        />
+      <TextField
+        label='Street'
+        name='street'
+        handlerFunction={this.handleStreetUpdate}
+        value={this.state.street}
+        />
+      <TextField
+        label='City'
+        name='city'
+        handlerFunction={this.handleCityUpdate}
+        value={this.state.city}
+        />
+      <TextField
+        label='State'
+        name='state'
+        handlerFunction={this.handleStateUpdate}
+        value={this.state.state}
+        />
+      <TextField
+        label='Zip'
+        name='zip'
+        handlerFunction={this.handleZipUpdate}
+        value={this.state.zip}
+        />
+      <TextField
+        label='Phone Number'
+        name='phoneNumber'
+        handlerFunction={this.handlePhoneNumberUpdate}
+        value={this.state.phoneNumber}
+        />
+      <TextField
+        label='Email'
+        name='email'
+        handlerFunction={this.handleEmailUpdate}
+        value={this.state.email}
+        />
+
+        <TextField
+          label='Website'
+          name='website'
+          handlerFunction={this.handleWebsiteUpdate}
+          value={this.state.website}
+          />
+
+        <section>
+                <div className="dropzone">
+                  <Dropzone onDrop={this.onDrop}>
+                    <p>Try dropping some files here, or click to select files to upload.</p>
+                  </Dropzone>
+                </div>
+                <aside>
+                  <h3>Dropped files</h3>
+                  <ul>
+                    {
+                      this.state.file.map(f => <li key={f.name}>{f.name} - {f.size} bytes</li>)
+                    }
+                  </ul>
+                </aside>
+              </section>
+
+        <input type="submit" className="submit-button button" value="Add Restaurant" />
+      </form>
+
+      </div>
+    );
+  }
+}
```

Comment:
Hold up....I am very confused. Why is there a RestuarantForm that renders a form AND a RestaurantFormContainer that renders a form?

Reply:
The Restaurant Form component, RestaurantForm.js is not being used after we refactored to reduce complexities in maintaining state. We need to remove it. Thanks for pointing it out